aviatrix/resource_aviatrix_gateway.go

```diff
@@ -1156,6 +1203,46 @@ func resourceAviatrixGatewayRead(d *schema.ResourceData, meta interface{}) error
 			d.Set("enable_monitor_gateway_subnets", false)
 			d.Set("monitor_exclude_list", "")
 		}
+
+		vpnGateway := &goaviatrix.Gateway{
+			GwName: gw.GwName,
+			VpcID:  gw.VpcID,
+		}
+
+		vpnConfigList, err := client.GetVPNConfigList(vpnGateway)
+		if err != nil && err != goaviatrix.ErrNotFound {
+			return fmt.Errorf("couldn't find vpn config list for gateway: %s", gw.GwName)
+		}
+
+		vpnConfigIdle := client.GetVPNConfig("Idle timeout", vpnConfigList)
+		if vpnConfigIdle != nil {
+			if vpnConfigIdle.Status == "enabled" {
+				idleTimeoutValue, err := strconv.Atoi(vpnConfigIdle.Value)
+				if err != nil {
+					return fmt.Errorf("couldn't convert vpn config value (idle timeout)")
+				}
+				d.Set("idle_timeout_value", idleTimeoutValue)
+			} else {
+				d.Set("idle_timeout_value", -1)
+			}
+		} else {
+			return fmt.Errorf("couldn't find vpn config (idle timeout) for gateway: %s", gw.GwName)
+		}
+
+		vpnConfigRenego := client.GetVPNConfig("Renegotiation interval", vpnConfigList)
+		if vpnConfigRenego != nil {
+			if vpnConfigRenego.Status == "enabled" {
+				renegoIntervalValue, err := strconv.Atoi(vpnConfigRenego.Value)
+				if err != nil {
+					return fmt.Errorf("couldn't convert vpn config value (renegotiation interval)")
+				}
+				d.Set("renegotiation_interval_value", renegoIntervalValue)
+			} else {
+				d.Set("renegotiation_interval_value", -1)
+			}
+		} else {
+			return fmt.Errorf("couldn't find vpn config (renegotiation interval) for gateway: %s", gw.GwName)
+		}
```

Comment:
This is just a nitpick not a big deal but these if statements can be simplified, check out this article: https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88 . Reducing the amount of indentation levels can increase readability of the code